show/main.py

```diff
@@ -192,6 +192,21 @@ def run_command(command, display_cmd=False):
     if rc != 0:
         sys.exit(rc)
 
+def run_command_save(command, display_cmd=False):
```

Comment:
This function is mostly redundant. Can we avoid adding another flavor of "run_command" here?

Also, the call to "show platform summary" is a lot of extra overhead, when this file contains all the info you need. For this one specifically, you could create a helper function to get the platform summary info, move the logic from "show platform summary" into that and call that helper function from both "show platform summary" and "show version".

Reply:
Created helper function: get_hw_info_dict to be used in "show version" and "show platform summary".  
Also removed function: run_command_save
and updated "show version" to use subprocess to get the info.  
Let me know if you need anything else.

Reply:
Retest this please